rflx/generator/allocator.py

```diff
@@ -206,7 +209,9 @@ def _create_init_pred(self, slots: List[NumberedSlotInfo]) -> UnitPart:
                             )
                             for slot in slots
                         ]
-                    ),
+                    )
+                    if slots
+                    else TRUE,
```

Comment:
That shouldn't be necessary, as `And` is already interpreted as `True` if there are no terms.

https://github.com/Componolit/RecordFlux/blob/f6a5c4e3736e02874866917e7e3fd48344fdcd29/rflx/ada.py#L140-L143.

Reply:
indeed, fixed!